src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,54 @@
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  border: 1px dotted #000;
+  border-radius: 50%;
+}
+
+.watch__axis {
+  z-index: 2;
+  margin: 45px;
+  height: 10px;
+  width: 10px;
+  background-color: #ffa500;
```

Comment:
is also a good practice to put all colors in separate variables, in a separate file

Reply:
It wasn't in the task but sure, will done)